src/mapml-viewer.js

```diff
@@ -986,7 +989,256 @@ export class HTMLMapmlViewerElement extends HTMLElement {
       }
     });
   }
+  matchMedia(query) {
```

Comment:
For documentation - all possible features:
- prefers-color-scheme (i.e. dark or light)
- prefers-lang (i.e. en, fr)
- map-projection (i.e. OSMTILE, CBMTILE)
- map-zoom (i.e. > or < Int)
- prefers-map-content
- map-top-left-easting (i.e. > or < Double)
- map-bottom-right-easting (i.e. > or < Double)
- map-bottom-right-northing (i.e. > or < Double)
- map-top-left-northing (i.e. > or < Double)